org/allPRs.ts

```diff
@@ -2,8 +2,23 @@ import {fail, warn, message, danger} from "danger"
 
 export const prSize = async () => {  
     // Warn when there is a big PR
-    if (danger.github.pr.additions + danger.github.pr.deletions > 500) {
-        warn("PR has more than 500 lines of code changing. Consider splitting into smaller PRs if possible.");
+    const excludedPaths = [
+        /\.xcodeproj\//,         // xcode project files
+        /\.xcassets\//,          // asset catalogs
+        /\.xcworkspace\//        // xcode workspace files
+    ];
+
+    const modifiedFiles = danger.git.modified_files;
+    const addedLines = danger.github.pr.additions;
+
+    // Check if all changed files are in excluded paths
+    const allFilesExcluded = modifiedFiles.every(file => 
+        excludedPaths.some(pattern => pattern.test(file))
+    );
+
+    // Only warn if there are non-excluded files and additions are >= 500
+    if (!allFilesExcluded && addedLines >= 500) {
```

Comment:
@aataraxiaa the way I read this is that if there's any not excluded file, this check will run and evaluate the sum of the diff in the PR, therefore including the changes in the excluded files. 
Is this your desired behaviour?

Reply:
Thanks @loremattei! No - what I had originally was not the desired behavior. We don’t want to include the excluded files in the diff count. I’ve updated the code and tests now.